frontend/app/templates/admin/content-approval.hbs

```diff
@@ -0,0 +1,231 @@
+<div class="m-3">
+  <h1 class="title">
+    {{t "admin.content_approval.titles.content_approval"}}
+  </h1>
+  <hr />
+  <div class="row info mb-3">
+    <div class="col-lg-3">
+      <div class="card h-100">
+        <div
+          class="card-body d-flex flex-column justify-content-center align-items-start"
+        >
+          <p>
+            <strong>
+              {{t 'admin.content_approval.labels.content_approval_prompt' greeting=this.greeting numOfPending=this.numOfPending}}
+            </strong>
+          </p>
+          <button class="btn btn-primary" type="button">
+            {{t 'admin.content_approval.buttons.start_approval'}}
+          </button>
+        </div>
+      </div>
+    </div>
+    <div class="col-lg-2">
+      <div class="card h-100">
+        <div
+          class="card-body d-flex flex-column align-items-center text-center"
+        >
+          <img src="/images/chapters-created.png" alt="" role="presentation" />
+          <p class="m-0">
+            {{t 'admin.content_approval.labels.pending_approval'}}
+          </p>
+          <p class="status-count m-0">
+            {{this.numOfPending}}
+          </p>
+        </div>
+      </div>
+    </div>
+    <div class="col-lg-2">
+      <div class="card h-100">
+        <div
+          class="card-body d-flex flex-column align-items-center text-center"
+        >
+          <img src="/images/completed-55.png" alt="" role="presentation" />
+          <p class="m-0">
+            {{t 'admin.content_approval.labels.approved_chapters'}}
+          </p>
+          <p class="status-count m-0">
+            {{this.numOfApproved}}
+          </p>
+        </div>
+      </div>
```

Comment:
on desktop the top 3 cards look like they are fighting for width. 
Is it possible to increase their width by an extra 1 column?